api/oc_message.c

```diff
@@ -44,29 +47,29 @@ message_deallocate(oc_message_t *message, struct oc_memb *pool)
 #if defined(OC_DYNAMIC_ALLOCATION) && !defined(OC_INOUT_BUFFER_SIZE)
```

Comment:
`defined(OC_DYNAMIC_ALLOCATION) && !defined(OC_INOUT_BUFFER_SIZE)` -> `OC_HAS_FEATURE_MESSAGE_SHRINK_BUFFER`

Seems that there are several places with the old defines. I would switch it to the new feature macro for consistency.

Reply:
Sure - fixed